### ISSUE TITLE
Performance of Stan. Dogs

### ISSUE THREAD

**Issue by xjiangmed** [FightingFighting/GPS#2]
Thanks for sharing the code! 

I ran your code on the FGVC dataset and followed the default training scripts. The performance I achieved on the NA-Birds, Oxford Flowers, and Stanford Cars datasets was comparable to the results in your paper. After adjusting the learning rate, I also achieved comparable performance on the CUB-2011 dataset.

However, when it comes to the Stanford Dogs dataset, despite adjusting the learning rate and times_para, I could only achieve a maximum performance of 88.82. I would appreciate any suggestions regarding the best hyperparameter configuration for this dataset.

**Comment by FightingFighting** [comment]
Hi @xjiangmed , thank you for your interest in our work. Actually, in order to compare with VPT, the Dog task is implemented base on VPT code base. Maybe you could try to use their code and implement our method in it.

**Comment by xjiangmed** [comment]
@FightingFighting  Thanks for your reply. I will try to implement it based on VPT.

**Comment by FightingFighting** [comment]
@xjiangmed yep, it should be easy to implement our method in VPT code. just let me know if you have any problems.

**Comment by xjiangmed** [comment]
@FightingFighting Can you provide the training scripts for the Stan. Dogs dataset on the swin and convnext backbones? Is it also based on the VPT code? I hope to reproduce the results of these two backbones. Thanks for your help.

**Comment by FightingFighting** [comment]
Hi @xjiangmed they should be implemented in ssf code. I am kind of busy these days and will upload it in two weeks.

**Comment by FightingFighting** [comment]
@xjiangmed I just checked. For the Stan. Dogs, we also implemented it in the VPT code.

**Comment by xjiangmed** [comment]
Thanks for your help!
